README.md

```diff
@@ -61,20 +61,21 @@ A template of the `licenses.json` file can be found in the repository at `src/ma
    # Used license scanner: scanoss (at the moment, only this scanner is supported)
     scanner=scanoss
 
-   # Path to the 'licenses.json' file which contains information about permitted,
-   # restricted and prohibited licenses. This file should be filled according to
-   # the template which could be found at 'src/main/resources/licenses.json'
-    license_filepath=
-
    # Used license conflicts source:
-   # > option "json": take conflicts from 'licenses.json' (should be filled manually
-   # according to the template at 'src/main/resources/licenses.json')
+   # > option "db": take conflicts from MySQL database - 'license_conflicts' table (should be filled manually
+   # according to the example at 'src/main/resources/database_dump.sql')
    # > option "scanner": take conflicts from the scanner response
-    license_conflict=json
+    license_conflict=db
+
+   # DB configuration (URL, username and password) - example
+   ...
+   spring.datasource.url=jdbc:mysql://localhost:3306/lpvs?useUnicode=true&characterEncoding=UTF-8&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
```

Comment:
It seems too long and would be very helpful if you consider the MySQL variables in terms of docker-compose environments.
Please see an [example](https://github.com/sameersbn/docker-redmine/blob/5880a5693d4632cd6e2c54c14c13c93b125d9022/docker-compose-mysql.yml#L21L27).

Reply:
Docker compose update will be included into the next PR.
Long string of DB setup is just an example. Could  be shorten (remove everything after ? sign).

Reply:
Yes. It would be nice to be able to remove the text after the question mark.

Reply:
> It seems too long and would be very helpful if you consider the MySQL variables in terms of docker-compose environments. Please see an [example](https://github.com/sameersbn/docker-redmine/blob/5880a5693d4632cd6e2c54c14c13c93b125d9022/docker-compose-mysql.yml#L21L27).

Added all variables that need to be filled from application.properties to docker-compose.yml for convience. 
Commented these variables in application.properties.